internal/spokes/spokes.go

```diff
@@ -909,3 +909,13 @@ func sideBandBufSize(capabilities pktline.Capabilities) int {
 	}
 	return 999
 }
+
+func isHex(s string) bool {
+	for i := 0; i < len(s); i++ {
+		c := s[i]
+		if !((c >= '0' && c <= '9') || (c >= 'a' && c <= 'z')) {
```

Comment:
```suggestion
		if !((c >= '0' && c <= '9') || (c >= 'a' && c <= 'f')) {
```

Or do you intend this to be closer to alpha-numeric?

Reply:
Will case ever matter? `|| (c >= 'A' && c <= 'F')`?

Reply:
We won't want to allow uppercase here because that makes pack names not deterministic.  Git never produces those, and we shouldn't accept them either.

Reply:
This was supposed to be lowercase hex only (`0-9a-f`). This commit is no longer a part of this branch. It was shipped as a part of #44 (f2f1510f0b99c9bc4d255a01a8f00537f683b077).